src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -13702,11 +13765,25 @@ int Assembler::vex_prefix_and_encode(int dst_enc, int nds_enc, int src_enc, VexS
 }
 
 int Assembler::evex_prefix_and_encode_ndd(int dst_enc, int nds_enc, int src_enc, VexSimdPrefix pre, VexOpcode opc,
-                           InstructionAttr *attributes, bool no_flags) {
+                           InstructionAttr *attributes, bool no_flags, bool use_prefixq) {
+  // Demote RegRegReg instructions
+  if (!no_flags && dst_enc == nds_enc) {
+    return use_prefixq? prefixq_and_encode(dst_enc, src_enc) : prefix_and_encode(dst_enc, src_enc);
```

Comment:
Nit pick, need space before ? as below:
use_prefixq ? prefixq_and_encode

Reply:
Please see the space fixed in the updated code.